src/topp/FileConverter.cpp

```diff
@@ -444,6 +481,104 @@ class TOPPFileConverter :
           s.getInstrumentSettings().setScanMode(InstrumentSettings::SRM);
         }
       }
+
+      if (store_im == "multiple")
+      {
+        bool has_im = false;
```

Comment:
maybe we could add a helper function getIMType() that returns an enum IM_NONE, IM_STACKED_ ...

Currently, you only specify how to store it right? Is the input format always the same? Because if not I don't know if the check for dirft time / data arrays work for every input combination (I am probably missing something here ;) ).

Reply:
> Currently, you only specify how to store it right? Is the input format always the same?

yes, this currently only works to convert from one format to the other and you have to know which format your data is in, otherwise it wont work